### PR TITLE
Feature/m1 support

### PR DESCRIPTION
**Pull request by atoennis**
**Attached Story/Issue**
Contributes to https://dowjones.atlassian.net/browse/WSJMOBILE-351

**Description**
Added M1 support.  Doing so required a new fork of this repo in order to replace the libwebp library with an xcframework.  Unfortunately, I couldn't find a way to pull in the xcframework version of libwebp via Carthage, instead it was manually downloaded from https://developers.google.com/speed/webp/download

**Changes**
- Removed references to all .frameworks (WebPDemux and WebPDecoder)
- Added .xcframeworks (WebPDemux and WebPDecoder)
- Updated the Cartfile to remove the reference to libwebp (since it was manually added)

### CODE REVIEW

File: WebPDecoder.xcframework/ios-arm64_armv7_armv7s/Headers/WebPDecoder/decode.h [hunk 1]
```diff
@@ -453,7 +453,7 @@ struct WebPDecoderOptions {
   int scaled_width, scaled_height;    // final resolution
   int use_threads;                    // if true, use multi-threaded decoding
   int dithering_strength;             // dithering strength (0=Off, 100=full)
-  int flip;                           // flip output vertically
+  int flip;                           // if true, flip output vertically
```
Comment (demosten) [inline WebPDecoder.xcframework/ios-arm64_armv7_armv7s/Headers/WebPDecoder/decode.h:456]: no need to specify, true is different in C/ObjC world but it is ok

File: WebPDecoder.xcframework/ios-arm64_i386_x86_64-simulator/Headers/WebPDecoder/decode.h [hunk 2]
```diff
@@ -453,7 +453,7 @@ struct WebPDecoderOptions {
   int scaled_width, scaled_height;    // final resolution
   int use_threads;                    // if true, use multi-threaded decoding
   int dithering_strength;             // dithering strength (0=Off, 100=full)
-  int flip;                           // flip output vertically
+  int flip;                           // if true, flip output vertically
```
Comment (demosten) [inline WebPDecoder.xcframework/ios-arm64_i386_x86_64-simulator/Headers/WebPDecoder/decode.h:456]: Same as above

File: Carthage/Checkouts/PINCache/Source/PINDiskCache.m [hunk 3]
```diff
@@ -17,7 +17,7 @@
 #import <PINOperation/PINOperation.h>
 #endif
 
-#define PINDiskCacheLogError(error) if (error) { NSLog(@"%@ (%d) ERROR: %@", \
+#define PINDiskCacheError(error) if (error) { NSLog(@"%@ (%d) ERROR: %@", \
```
Comment (demosten) [inline Carthage/Checkouts/PINCache/Source/PINDiskCache.m:20]: I don't really get why was this renamed

**Review by demosten — APPROVED** [review]
I've left some comments. You can answer/fix them or not

File: WebPDecoder.xcframework/ios-arm64_armv7_armv7s/Headers/WebPDecoder/decode.h (hunk 1 above)
Comment (atoennis) [reply WebPDecoder.xcframework/ios-arm64_armv7_armv7s/Headers/WebPDecoder/decode.h:456]: Good point.  I just realized I forgot to mention in the description that as part of this effort libwebp was updated from 1.1.0 to 1.2.2.  I believe the comment modification was due to those version changes

File: Carthage/Checkouts/PINCache/Source/PINDiskCache.m (hunk 3 above)
Comment (atoennis) [reply Carthage/Checkouts/PINCache/Source/PINDiskCache.m:20]: Yeah, great question.  I'm not sure either.  I know I updated libwebp but PINCache was unchanged at 3.0.3 but this code was changed after I ran a Carthage update.
